MAVProxy/modules/mavproxy_magical/__init__.py

```diff
@@ -0,0 +1,194 @@
+# Copyright (C) 2016  Intel Corporation. All rights reserved.
+#
+# This file is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This file is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+Compass calibration helper. This is adapted from Ardupilot's local MAVProxy
+module magcal_graph. Some people read it "magical_graph" at the time of
```

Comment:
While cute, @tridge might want something a little more descriptive for this module :-)